WordPress/Classes/ViewRelated/Blog/Blog Dashboard/BlogDashboardViewController.swift

```diff
@@ -21,21 +25,33 @@ final class BlogDashboardViewController: UIViewController {
 
     private lazy var dataSource: DataSource = {
         return DataSource(collectionView: collectionView) { collectionView, indexPath, identifier in
-            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: QuickLinksHostCell.defaultReuseID, for: indexPath) as? QuickLinksHostCell
-            cell?.hostedView = QuickLinksView(title: self.quickLinks[indexPath.item])
+            if identifier == "Quick Links" {
+                let cell = collectionView.dequeueReusableCell(withReuseIdentifier: QuickLinksHostCell.defaultReuseID, for: indexPath) as? QuickLinksHostCell
+                cell?.hostedView = QuickLinksView(title: self.quickLinks[indexPath.item])
+                return cell
+            }
+
+            let cell = collectionView.dequeueReusableCell(withReuseIdentifier: DashboardCardCell.defaultReuseID, for: indexPath) as? DashboardCardCell
+            cell?.configure(self, blog: self.blog!)
```

Comment:
What do you think of using a switch statement on indexPath.section?

Reply:
Done in ed059b2d98d56e86281b09d25d82178cbafd8ed8